res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -38,7 +42,7 @@ limitations under the License.
         }
     }
 
-    .mx_BaseCard_back ~ .mx_ThreadPanel__header {
+    .mx_BaseCard_back~.mx_ThreadPanel__header {
```

Comment:
this change seems unrelated (and more complex to follow - will add to our style guide)

Reply:
Gahh, sorry, my VSCode is going crazy!